Add roadmap (#64)
grafana/xk6-disruptor: README.md

Check the [get started guide](/docs/01-get-started/01-get-started.md) for instructions on how to install and use `xk6-disruptor`.

> The [Roadmap](/ROADMAP.md) presents the goals of the project for the month to come in terms of new functionalities and enhancements.

Reviewer: ```suggestion
The [Roadmap](/ROADMAP.md) presents the project's goals for the coming months regarding new functionalities and enhancements.
```
